test/language/generators/generator.expression.no-name.js

```diff
@@ -0,0 +1,19 @@
+// Copyright 2015 Cubane Canada, Inc.  All rights reserved.
+// See LICENSE for details.
+
+/*---
+info: >
+    Generator can be declared with GeneratorExpression syntax
+es6id: 14.4
+author: Sam Mikes
+description: can create generator function expressions (no name)
+---*/
+
+var f = function *(a) { yield a+1; return; };
+
+assert.sameValue(f.name, '');
```

Comment:
I believe the name of this generator should be 'f', cf.

https://people.mozilla.org/~jorendorff/es6-draft.html#sec-variable-statement-runtime-semantics-evaluation
https://people.mozilla.org/~jorendorff/es6-draft.html#sec-isanonymousfunctiondefinition
https://people.mozilla.org/~jorendorff/es6-draft.html#sec-generator-function-definitions-static-semantics-isfunctiondefinition